bin/check-memory-percent.rb

```diff
@@ -3,4 +3,4 @@
 bin_dir = File.expand_path(File.dirname(__FILE__))
 shell_script_path = File.join(bin_dir, File.basename($PROGRAM_NAME, '.rb') + '.sh')
 
-exec shell_script_path, *ARGV
+exec shell_script_path, *ARGV, :unsetenv_others => true
```

Comment:
This is probably ok though I have not inspected the scripts to see if there is anything we need.